aries-site/src/layouts/content/DocsPageHeader.js

```diff
@@ -27,11 +27,13 @@ export const DocsPageHeader = ({ title, topic, render }) => {
           </Link>
         )
       }
-      subtitle={<SubsectionText>{page.description}</SubsectionText>}
+      subtitle={
+        <SubsectionText accessibility={page.accessibility}>
```

Comment:
The accessibility tag feels separate from the subtitle, wondering if there is a different way to structure this. If there isn't we can leave as it, but just thought I would ask

Reply:
yeah it does seem a little weird agree there however this is built with PageHeader and doesnt account for `children` so had to put it within the `subtitle`